### PR TITLE
add structure and example

### PR DESCRIPTION
**Pull request by mokkin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +26,14 @@ All boolean fields default to null if not specified
 | calendar-resource:resources:list | List all resources | | | |
 | calendar-resource:resource:delete | Delete a resource and anything that belongs to them | `resource_type`* `id`* | | |
 
+### Example for creating a room inspired by the [post from jos](https://help.nextcloud.com/t/22-beta-5-is-here-help-test/118560):
+
+```
+php occ calendar-resource:building:create "Nextcloud office Berlin" "Gerichtstraße 23, 13347 Berlin, Germany" ""
```
Comment (ChristophWurst) [inline README.md:32]: this information were accidental published. Please change it to some fake data

File: README.md [hunk 2]
```diff
@@ -29,9 +29,9 @@ All boolean fields default to null if not specified
 ### Example for creating a room inspired by the [post from jos](https://help.nextcloud.com/t/22-beta-5-is-here-help-test/118560):
```
Comment (ChristophWurst) [inline README.md:29]:
```suggestion
### Example for creating a room
```

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
This can go too :)

**Comment by mokkin** [comment]
Wouldn't it be better then to ask @jospoortvliet for changing his original post in the forum?

**Comment by miaulalala** [comment]
> Wouldn't it be better then to ask @jospoortvliet for changing his original post in the forum?

Posts change or might get deleted, the URL might be outdated etc. Would you mind removing the reference?

File: README.md (hunk 1 above)
Comment (jospoortvliet) [reply README.md]: The nextcloud office address isn't secret, but certainly the email addresses should be fake-ified ;-)

**Comment by mokkin** [comment]
No matter if it is important for. Should be fine now.

**Comment by miaulalala** [comment]
Thanks so much!

**Review by miaulalala — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**